lib/network_utils.pm

```diff
@@ -90,8 +90,13 @@ sub recover_network {
 }
 
 sub ifc_exists {
-    my ($ifc) = @_;
-    return !script_run('ip link show dev ' . $ifc);
+    my ($ifc, $check_is_up) = @_;
```

Comment:
not sure but maybe : 
```suggestion
    my ($self, $ifc, $check_is_up) = @_;
```

Reply:
I'm not sure. The other functions don't use $self and we don't need it in the code of the function.

Reply:
you right https://perlmaven.com/methods-functions-and-subroutines-in-perl , when there is no calls `$self->ifc_exist()` we should not use $self as first arg . As I said "I am not sure" :)